bulbs/special_coverage/api.py

```diff
@@ -7,7 +7,6 @@
 
 
 class SpecialCoverageViewSet(viewsets.ModelViewSet):
-    queryset = SpecialCoverage.objects.all()
```

Comment:
ah, i was just trying something out, so i can put this back. theoretically, using `get_queryset()` would be useful if we wanted to dynamically adjust the queryset.